docs/tutorials/create-waggle.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 4
+---
+
+# Building your own Waggle Device
+
+:::
+Note: This tutorial is work in progress and is made available here for early preview.
+:::
+
+Do you ever look into the sky and wonder how will the cloud move over time? Do you ever want to know if a wildfire is possibly to start in a forest? If you would like to build, design and deploy a software that could answer your questions above, then Waggle is the right choice for you. Waggle is an edge computing wireless sensor device, say, you can define what the device will be detecting by building a software on it.
+
+If you would like to gain more understandings and share your own observation, this tutorial will give you a guide of how to register your Waggle device to the Sage network. After registering, you can publish your observation and see others data on [Sage Portal](https://portal.sagecontinuum.org/apps/explore).
+
+
+## Potential Use Cases
+
+You might be ...
+- a professor that wants to introduce Waggle device to your students and plans to instruct students to make 20+ waggle devices
+- someone interested in Waggle and making an app but doesnt have access to production nodes
+- a new user of waggle network that wants to use a new sensor (i.e. a new camera, a new bat singal detector, a new custom sensor they built)
+
+If you have your device ready, you could follow the instructions in [Access Waggle sensors](./access-waggle-sensors.md) to start defining the edge computing software on the device. If you don't, you could use a Sage Edge Computing kit to easy set up a hardware platform for development.
```

Comment:
I am going to suggest here the rest of the document. I would like to take everything you wrote and re-arrange it like the following:

```
## Getting Started

To get started in boot-strapping your Waggle Edge Computing kit you can follow the instructions for the various supported platforms on the [node-platforms](https://github.com/waggle-sensor/node-platforms) GitHub page.

> The currently only supported platform is the [NVidia nano](https://github.com/waggle-sensor/node-platforms/tree/main/nvidia-nano) but more platforms will be added in the future. In fact, if you would like to add support for other platforms go ahead and submit a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to [node-platforms](https://github.com/waggle-sensor/node-platforms).

### Registering Your Device

During the bootstrapping process you will have the option to register your device within the [web portal](https://portal.sagecontinuum.org/apps/explore). It is highly recommended to register your device, as this enables all the core [WES tools](LINK HERE) to be automatically downloaded, enabling the edge app development and run-time environment. Additionally, this enables your edge apps to publish data to the development [Beehive](LINK HERE), accessible to cloud-based analysis tools and workflow frameworks.

To register your device navigate to the [My Devices Web Portal](https://portal.sagecontinuum.org/my-devices). Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. A "registration zip" file will be generated and available for download. Then follow the instructions for [your device](https://github.com/waggle-sensor/node-platforms) to load the registration keys.

> You may register as many times as you want. But note that each registration key has a short expiration time and should be used shortly after generation.
```

Then lets add a "now you a ready to develop your edge apps and/or introduce new sensors to the Waggle platform" conclusion paragraph with links to the other tutorials.